### PR TITLE
abdalla/ redirect /en to /

### PR DESCRIPTION
**Pull request by abdalla-binary**
# Description

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -60,9 +67,4 @@ exports.onCreateWebpackConfig = ({ actions, getConfig }) => {
             modules: [path.resolve(__dirname, 'src'), 'node_modules'],
         },
     })
-}
-
-exports.createPages = ({ actions }) => {
-    const { createRedirect } = actions //actions is collection of many actions - https://www.gatsbyjs.org/docs/actions
-    createRedirect({ fromPath: '/en', toPath: '/', redirectInBrowser: true, isPermanent: true })
-}
+}
```
Comment (abdalla-binary) [inline gatsby-node.js:70]:
```suggestion
```

File: gatsby-node.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ exports.onCreatePage = ({ page, actions }) => {
             translations_cache[lang] = translation_json
         }
 
-        return createPage({
+        const curruntPage = createPage({
```
Comment (kevinw-binary) [inline gatsby-node.js:39]: shouldn't it be `currentPage`?

File: gatsby-node.js [hunk 3]
```diff
@@ -49,6 +49,14 @@ exports.onCreatePage = ({ page, actions }) => {
...
             },
         })
+
+        if (is_default) {
+            const enPath = `/en${localized_path.slice(0, -1)}`
+            createRedirect({ fromPath: enPath, toPath: localized_path, redirectInBrowser: true, isPermanent: true})
+            createRedirect({ fromPath: `${enPath}/`, toPath: localized_path, redirectInBrowser: true, isPermanent: true})
+        }
+
+        return curruntPage
```
Comment (kevinw-binary) [inline gatsby-node.js:59]: same as here `currentPage`

File: gatsby-node.js [hunk 4]
```diff
@@ -49,6 +49,14 @@ exports.onCreatePage = ({ page, actions }) => {
                 pathname: localized_path,
             },
         })
+
+        if (is_default) {
+            const enPath = `/en${localized_path.slice(0, -1)}`
```
Comment (mustofa-binary) [inline gatsby-node.js:54]: please use snake_case for non function or class variable 🙏

**Review by kevinw-binary — APPROVED** [review]

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**